contracts/plugins/assets/ATokenFiatCollateral.sol

```diff
@@ -2,32 +2,30 @@
 pragma solidity 0.8.9;
 
 import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
-import "@openzeppelin/contracts/utils/math/Math.sol";
 import "contracts/libraries/Fixed.sol";
 import "contracts/plugins/assets/AbstractCollateral.sol";
 
-// ==== External ====
-
-// External interfaces from: https://git.io/JX7iJ
+// This interface is redundant with contracts/plugins/aave/IStaticAToken, but it's compiled with
+// a different version
```

Comment:
hehe -- I claim that you don't really need to apologize for abstracting a contract with an interface. :)

Reply:
The apology is for duplicate interfaces in the same repo, for the same contract. Leaving a slightly tweaked version of the note